src/test/java/uk/gov/hmcts/reform/sendletter/tasks/UploadLettersTest.java

```diff
@@ -0,0 +1,58 @@
+package uk.gov.hmcts.reform.sendletter.tasks;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.test.context.junit4.SpringRunner;
+import uk.gov.hmcts.reform.sendletter.LocalSftpServer;
+import uk.gov.hmcts.reform.sendletter.SampleData;
+import uk.gov.hmcts.reform.sendletter.entity.Letter;
+import uk.gov.hmcts.reform.sendletter.entity.LetterRepository;
+import uk.gov.hmcts.reform.sendletter.entity.LetterState;
+import uk.gov.hmcts.reform.sendletter.helper.FtpHelper;
+import uk.gov.hmcts.reform.sendletter.services.LetterService;
+
+import java.io.File;
+import java.util.UUID;
+import javax.persistence.EntityManager;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(SpringRunner.class)
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+@DataJpaTest
+public class UploadLettersTest {
+
+    @Autowired
+    LetterRepository repository;
+
+    @Autowired
+    private EntityManager entityManager;
+
+    @Test
+    public void uploads_pdfs_to_sftp() throws Exception {
```

Comment:
either update test name, or (probably better) create a separate one for checking the db update part